Cubical/Categories/Sets.agda

```diff
@@ -9,9 +9,9 @@ open import Cubical.Categories.Category
 module _ ℓ where
   SET : Precategory (ℓ-suc ℓ) ℓ
```

Comment:
`SetPrecat`?

Reply:
Hm isn't SET sort of like the standard name for this category though?